mne/time_frequency/tfr.py

```diff
@@ -1534,6 +1541,24 @@ def __repr__(self):  # noqa: D105
         s += ', ~%s' % (sizeof_fmt(self._size),)
         return "<EpochsTFR  |  %s>" % s
 
+    def __abs__(self):
+        out = self.copy()
+        np.abs(out.data, out=out.data)
```

Comment:
using out.data when you can from complex to float seems wrong.

do I miss something?

Reply:
argh, no you're right

Reply:
fixed